apps/server/src/shared/domain/entity/lesson.entity.ts

```diff
@@ -17,6 +17,7 @@ export interface LessonProperties {
 	position?: number;
 	contents: ComponentProperties[] | [];
 	materials?: Material[];
+	tasks?: Task[];
```

Comment:
This is not correct. The lesson do not know the tasks. The task know his parents, but we do not have bidirectional relationships in the database.

Reply:
Do we have a good way to get the tasks for a lesson? does a proper repo call exist already?

Reply:
The lesson entity has a tasks property and that is not from us (Line 110).

Reply:
I removed the tasks because they are not necessary for the export. But I am also interested in the answer to Patrick's comment.